Apps/Encounter/src/Models/ODR/MSPVisitHistory.cs

```diff
@@ -26,13 +26,13 @@ public class MSPVisitHistory : ODRHistoryWrapper
         /// <summary>
         /// Gets or sets the ODRHistoryQuery for the MSPVisitHistory integration.
         /// </summary>
-        [JsonPropertyName("getMSPVisitHistoryRequest")]
+        [JsonPropertyName("getMspVisitHistoryRequest")]
```

Comment:
good catch. We didn't account for the capitalization on the other side of the pipe